src/Catalog.API/Apis/CatalogApi.cs

```diff
@@ -1,4 +1,11 @@
-﻿using Microsoft.AspNetCore.Http.HttpResults;
+﻿using System.ComponentModel;
```

Comment:
Do we need some of the usings here like `Grpc.Core` and `Microsoft.VisualBasic`?